<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Data_report</class>
 <widget class="QMainWindow" name="Data_report">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1328</width>
    <height>792</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/****#Data_report{
background-image: url(:/img/room.jpg);
}*/</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>110</y>
      <width>531</width>
      <height>591</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/*background-color: rgb(231, 231, 231);</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>461</width>
       <height>281</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>28</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"> QTabWidget{
                background-color:transparent;
}
                QTabWidget::pane{
                    border:2px;
                }
                QTabWidget::tab-bar{
                       background:transparent;
						subcontrol-position:center;
                }
                QTabBar::tab{
                    background:rgb(14, 106, 175);
                    color:white;
					min-width:70px;
					min-height:55px;
                }
                QTabBar::tab:hover{
                    background:rgb(255, 255, 255, 100);
                color:black;
                }
                QTabBar::tab:selected{
                    border-color: black;
                    background:red;
                    color:white;
                }
</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>传感器报表</string>
      </attribute>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>24</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>报表日期：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>140</y>
         <width>191</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>24</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>传 感 器：</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>30</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>13</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <item>
        <property name="text">
         <string>2021年1月27日</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="com_sensor">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>150</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <item>
        <property name="text">
         <string>SCSQ210124001-1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SCSQ210124001-2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SCSQ210124001-3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SCSQ210124001-4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SCSQ210124001-5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SCSQ210124001-6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SCSQ210124002-1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SCSQ210124002-2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SCSQ210124002-3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SCSQ210124002-4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SCSQ210124002-5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SCSQ210124002-6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SCSQ210124003-1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SCSQ210124003-2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SCSQ210124003-3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SCSQ210124003-4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SCSQ210124003-5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SCSQ210124003-6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SCSQ210124004-1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SCSQ210124004-2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SCSQ210124004-3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SCSQ210124004-4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SCSQ210124004-5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SCSQ210124004-6</string>
        </property>
       </item>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>房间报表</string>
      </attribute>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>181</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>24</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>报表日期：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>150</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>24</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>请 选 择：</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_3">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>30</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <item>
        <property name="text">
         <string>2021年1月27日</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="com_room">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>150</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>360</y>
       <width>421</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>24</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox::title
{
color: rgb(255, 255, 255);
subcontrol-origin:margin;
subcontrol-position:top center;
padding:0 3px;
}</string>
     </property>
     <property name="title">
      <string>生成数据间隔</string>
     </property>
     <widget class="QRadioButton" name="radioButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">
QRadioButton{
    spacing: 2px;
    color: white;
}
QRadioButton::indicator {
    width:                  12px;
    height:                 12px;
    border-radius:          8px;
	
}
 
QRadioButton::indicator:checked {
	
    background-color:       green;
    border:                 2px solid white;
	
	color: rgb(255, 255, 255);
}
 
QRadioButton::indicator:unchecked {
    background-color:       white;
    border:                 2px solid white;
	color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="text">
       <string>3分钟</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>50</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">
QRadioButton{
    spacing: 2px;
    color: white;
}
QRadioButton::indicator {
    width:                  12px;
    height:                 12px;
    border-radius:          8px;
	
}
 
QRadioButton::indicator:checked {
	
    background-color:       green;
    border:                 2px solid white;
	
	color: rgb(255, 255, 255);
}
 
QRadioButton::indicator:unchecked {
    background-color:       white;
    border:                 2px solid white;
	color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="text">
       <string>15分钟</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_3">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>50</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">
QRadioButton{
    spacing: 2px;
    color: white;
}
QRadioButton::indicator {
    width:                  12px;
    height:                 12px;
    border-radius:          8px;
	
}
 
QRadioButton::indicator:checked {
	
    background-color:       green;
    border:                 2px solid white;
	
	color: rgb(255, 255, 255);
}
 
QRadioButton::indicator:unchecked {
    background-color:       white;
    border:                 2px solid white;
	color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="text">
       <string>30分钟</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="btn_sure">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>500</y>
       <width>431</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
color: rgb(255, 255, 255);
 border:5px outset rgb(39,118,129);
 background-color: qconicalgradient(cx:0.5, cy:0.5, angle:180, stop:0.49999 rgba(46, 138, 149, 255), stop:0.50001 rgba(55, 158, 169, 255));
border-radius:10px;
padding:2px 4px;
}

QPushButton:hover{
	
	color: rgb(0, 0, 0);
    border:5px outset gray;
    background-color: qconicalgradient(cx:0.5, cy:0.5, angle:180, stop:0.49999 rgba(188, 188, 188, 255), stop:0.50001 rgba(208, 208, 208, 255));
    border-radius:10px;
	position:relative; top:1px; left:1px
}

QPushButton:pressed{
    border:3px inset gray;
    background-color: qconicalgradient(cx:0.5, cy:0.5, angle:180, stop:0.49999 rgba(208, 208, 208, 255), stop:0.50001 rgba(188, 188, 188, 255));
	border-bottom: 1px;
    border-radius:10px;
}</string>
     </property>
     <property name="text">
      <string>生成报表</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
